src/components/Dashboard/Analytics/AnalyticsUtils.js

```diff
@@ -0,0 +1,121 @@
+// Barcharts
+
+export const barChartOptions = {
+    plugins: {
+        legend: {
+            position: "top",
+            display: false,
+        },
+        title: {
+            display: false,
+            text: "Activity",
+        },
+    },
+    maintainAspectRatio: false,
+    responsive: true,
```

Comment:
you use the same structure with different values afterwards make it a general function.